Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:16-alpine
+WORKDIR /app
+
+# Since 0.14.0 release of html-template chromium is needed for pdf generation
+ENV PUPPETEER_EXECUTABLE_PATH /usr/bin/chromium-browser
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
+# Since 0.30.0 release Git is supported and required as a dependency
+# Since 0.14.0 release of html-template chromium is needed for pdf generation.
+# More custom packages for specific template should not be added to this dockerfile. Instead, we should come up with some extensibility solution.
+RUN apk --update add git chromium && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /var/cache/apk/*
+
+# Installing latest released npm package
+RUN npm install --ignore-scripts -g @asyncapi/cli
+
+ENTRYPOINT [ "asyncapi" ]
```

Comment:
```suggestion
FROM node:16-alpine

# Create a non-root user
# Addressing sonar cloud: The node image runs with root as the default user.
RUN addgroup -S myuser && adduser -S myuser -G myuser

WORKDIR /app

# Since 0.14.0 release of html-template chromium is needed for pdf generation
ENV PUPPETEER_EXECUTABLE_PATH /usr/bin/chromium-browser
ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true

# Since 0.30.0 release Git is supported and required as a dependency
# Since 0.14.0 release of html-template chromium is needed for pdf generation.
# More custom packages for a specific template should not be added to this Dockerfile. Instead, we should come up with some extensibility solution.
RUN apk --update add git chromium && \
    rm -rf /var/lib/apt/lists/* && \
    rm /var/cache/apk/*

# Installing the latest released npm package
RUN npm install --ignore-scripts -g @asyncapi/cli

# Switch to the non-root user
USER myuser

ENTRYPOINT [ "asyncapi" ]
```

this should solve the security issue

Reply:
Yeah i tried the same method but then while running the image it throws permission error
@derberg

Reply:
Found a fix!

Reply:
interesting, I tried it on local and there were no errors, anyway, I see all green now 🚀